src/ProjectDirectory.php

```diff
@@ -6,6 +6,11 @@ class ProjectDirectory
 {
     public static function find(/* string */ $cwd)
     {
+        $env = getenv('WHIPPET_PROJECT_DIRECTORY');
+        if ($env) {
+            return \Result\Result::ok(new self($env));
```

Comment:
Good question. I was thinking of this as an override. So you can just force whippet to use a particular directory without it asking any questions.

But I suppose I should probably make it use the normal logic.